### PR TITLE
LPS-178763 Integration Tests for Author and Current User Terms

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-178763

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/BaseNotificationTypeTest.java [hunk 1]
```diff
@@ -40,7 +75,95 @@ public class BaseNotificationTypeTest {
...
 	@BeforeClass
 	public static void setUpClass() throws Exception {
-		user = TestPropsValues.getUser();
+		user1 = TestPropsValues.getUser();
+
+		ListType prefixListType = listTypeLocalService.getListType(
+			"dr", "com.liferay.portal.kernel.model.Contact.prefix");
+
+		ListType suffixListType = listTypeLocalService.getListType(
+			"ii", "com.liferay.portal.kernel.model.Contact.suffix");
```
Comment (MarinhoFeliphe) [inline modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/BaseNotificationTypeTest.java:84]: @p-albuquerque replace with `Contact.class.getName() + ".preffix"`

File: modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/BaseNotificationTypeTest.java [hunk 2]
```diff
@@ -63,6 +186,86 @@ protected NotificationRecipientSetting createNotificationRecipientSetting(
...
+		if (listTypeId == 0) {
+			return StringPool.BLANK;
+		}
+
+		ListType listType = listTypeLocalService.getListType(listTypeId);
+
+		return listType.getName();
+	}
+
+	protected HashMap<String, Serializable> getObjectEntryValues() {
```
Comment (MarinhoFeliphe) [inline modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/BaseNotificationTypeTest.java:251]: @p-albuquerque rename to `randomObjectEntryValue()`

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
@p-albuquerque

I've made some changes (https://github.com/p-albuquerque/liferay-portal/pull/45) the last commit I'd like you take a look an try to apply, basicly I'm just concating all terms (objectEntry, author and currentUser) and asserting them in the same test.

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by p-albuquerque** [comment]
Resent in: #1899

**Pull request closed without merging**